app/views/articles/show.html.erb

```diff
@@ -1,2 +1,77 @@
-<h1>Articles#show</h1>
-<p>Find me in app/views/articles/show.html.erb</p>
+<div class="container">
+  <div class="row">
+
+    <div class="col-xs-8">
+
+      <h2><%= @article.title %></h2>
+      <p><em><%= @article.summary %></em></p>
+      <p>(£ <%= @article.price %> per day) </p>
+
+      <!-- Show URL/content only if accessible, i.e. owned or booked -->
+
+      <% accessible_articles = current_user.articles + current_user.today_booked_articles %>
+
+      <% if accessible_articles.include? @article %>
+        <%= link_to @article.url, @article.url %>
+      <% else %>
+        <p>You do not have access to this article</p>
+      <% end %>
+
+    </div>
+
+    <div class="col-xs-4">
+
+      <!-- Show booking form if not currently booked, or booking details if active booking -->
+      <% if current_user.today_booked_articles.include? @article %>
+        <%= render template: 'bookings/show' %>
+      <% else %>
+        <%= render template: 'bookings/new', article: @article, booking: @booking %>
+      <% end %>
+
+    </div>
+
+  </div>
+
+  <br>
+
+
+  <div class="row">
+    <div class="col-xs-12">
+      <h3>All reviews for <%= @article.title %></h3>
+    </div>
+  </div>
+
+  <br>
+
+  <% @article.reviews.each do |review| %>
+    <div class="row">
+
+      <div class="col-xs-3">
+        <% 1.upto(5) do |i| %>
+          <% if i <= review.rating %>
+            <i class="fa fa-star" aria-hidden="true"></i>
+          <% else %>
+            <i class="fa fa-star-o" aria-hidden="true"></i>
+         <% end %>
+        <% end %>
+      </div>
+
+
+      <div class="col-xs-9">
+        <p><em><%= review.content %></em></p>
+      </div>
+
+    </div>
+    <hr>
+
+  <% end %>
+
+  <div class="row">
+    <div class="col-xs-12">
+      <h4>Leave a review</h4>
+      <%= render template: 'reviews/new', article: @article, review: @review %>
+    </div>
+  </div>
+
+
```

Comment:
Sure - the booking form doesn't work so will need to fix it as well. We can discuss tomorrow